.github/workflows/release.yml

```diff
@@ -24,3 +24,30 @@ jobs:
           args: release --clean
         env:
           GITHUB_TOKEN: ${{secrets.GH_PAT}}
+  
+  docker:
+    runs-on: ubuntu-latest
+    needs: goreleaser
```

Comment:
any reason we need to wait for the `goreleaser` job? I don't see any hard dependency and we could save some time by executing these jobs in parallel